kratos/geometries/geometry.h

```diff
@@ -1869,12 +1869,14 @@ class Geometry
      * @brief Returns the local coordinates of a given arbitrary point
      * @param rResult The vector containing the local coordinates of the point
      * @param rPoint The point in global coordinates
+     * @param rConverged A boolean indicating whether the computation has converged
      * @return The vector containing the local coordinates of the point
      */
     virtual CoordinatesArrayType& PointLocalCoordinates(
-            CoordinatesArrayType& rResult,
-            const CoordinatesArrayType& rPoint
-            ) const
+        CoordinatesArrayType& rResult,
+        const CoordinatesArrayType& rPoint,
+        bool& rConverged
+        ) const
```

Comment:
`rResult` is passed in as reference in the arguments. It is also returned as a reference. This is dumb. Maybe now's a good time to fix this.

I'd return the `bool` instead of passing it by ref.

```cpp
virtual [[nodiscard]] bool PointLocalCoordinates(
        CoordinatesArrayType& rResult,
        const CoordinatesArrayType& rPoint
        ) const
```